src/redux/configureStore.js

```diff
@@ -1,29 +1,27 @@
+import { applyMiddleware, compose, createStore } from 'redux'
+import { syncHistory } from 'redux-simple-router'
 import thunk from 'redux-thunk'
 import rootReducer from './rootReducer'
-import {
-  applyMiddleware,
-  compose,
-  createStore
-} from 'redux'
 
-export default function configureStore (initialState) {
-  let createStoreWithMiddleware
-  const middleware = applyMiddleware(thunk)
+function withDevTools (middleware) {
+  const devTools = window.devToolsExtension
+    ? window.devToolsExtension()
+    : require('containers/DevTools').default.instrument()
```

Comment:
also fyi, we're using [babel-plugin-add-module-exports](https://github.com/59naga/babel-plugin-add-module-exports) so the default key is only necessary at runtime. if default exists, the plugin attaches a reference directly to module.exports during compilation... for some context, this is the output without plugin:

```js
exports.default = configureStore;
```
output with plugin:
```js
exports.default = configureStore;
module.exports = exports['default'];
```